Assets/Editor/MenuItems.cs

```diff
@@ -5,9 +5,9 @@
 namespace Editor
 {
     /// <summary>
-    /// Manager for editor menu items, etc.
+    /// Instantiates functions and windows to be run from the Unity editor menu.
     /// </summary>
-    public class MenuItems : MonoBehaviour
+    public class MenuItems : ScriptableObject
```

Comment:
I think the class can just be a static class and it should just work.

ScriptableObject may also cause problems but haven't checked.

Reply:
thanks, changed